_source/_posts/2020-07-23-introducing-jpaseto.md

```diff
@@ -14,7 +14,7 @@ image: blog/introducing-jpaseto/jpaseto-social.png
 type: awareness
 ---
 
-PASETO is a new security token format designed to be easy to use and free from the issues inherent with JSON Web Token (JWT) related specifications. Platform Agnostic SEcurity TOkens (PASETO) is a draft RFC spec created by [Scott Arciszewski](https://paragonie.com/). PASETO reduces the scope of the Javascript Object Signing and Encryption (JOSE) family of specs (of which JWT is a part of), while still providing the functions that secure applications need.  
```

Comment:
Javascript _could_ be JavaScript here, but JOSE is both referenced as:
* Javascript Object Signing and Encryption - lower case s
* JSON Object Signing and Encryption - which is redundant 

Thoughts?

Reply:
Upper case "S".